example/integration_tests/java-toolchain/sample/Main.java

```diff
@@ -0,0 +1,9 @@
+
+import org.graalvm.home.Version;
+
+
+public class Main {
+    public static void main(String args[]) {
+        System.out.println("Hello, GraalVM: " + Version.getCurrent().toString());
```

Comment:
`org.graalvm.home.Version` is only available via the GraalVM SDK, which ships embedded int he GraalVM JVM. so this test will not build _unless_ GraalVM is installed and working as the `tool_java_runtime_version`.